qemu/Dockerfile

```diff
@@ -104,7 +104,6 @@ RUN echo http://mirror.yandex.ru/mirrors/alpine/v3.7/community >> /etc/apk/repos
             mips64el-softmmu \
             moxie-softmmu \
             ppc-softmmu \
-            ppcemb-softmmu \
```

Comment:
The "qemu-system-ppcemb" target has been removed. "qemu-system-ppc" or "qemu-system-ppc64" should be used instead.

Reply:
Add a proof link please.

Reply:
https://wiki.qemu.org/ChangeLog/3.1#Incompatible_changes

Reply:
You have to remove it from README as well.